features/meta-theme-color.yml

```
@@ -0,0 +1,7 @@
+name: '<meta name="theme-color">'
+description: 'The "theme-color" `<meta>` name sets the preferred color to customize the display of the page or the surrounding browser user interface.'
```

To match the way we've phrased similar attribute-value pairs elsewhere (e.g., `alternative-style-sheets`):

```suggestion
description: The `name="theme-color"` attribute for the `<meta>` HTML element sets the preferred color to customize the display of the page or the surrounding browser user interface.
```